### ISSUE TITLE
@icloud.com not arriving 

### ISSUE THREAD

**Issue by MaizeStarch** [PHPMailer/PHPMailer#1144]
I have the same problem like this: https://github.com/PHPMailer/PHPMailer/issues/1140

I can send emails to @gmail and @hotmail and they arrive sucessfully, but they don't it in @icloud.com

However, unlike the previous question, I decided to investigate and found this, It's a tool to analyze the email’s delivery route from my hosting provider.

it says that the message was sucessfully sent, but it never arrives anywhere.
![test](https://user-images.githubusercontent.com/20288677/30044708-a079a79a-91c4-11e7-81f5-3e334ed41a50.png)

If I send from other emails (including randoms emails), shows the status as expected. (Errors, warnings, or sucess like this screenshot)

What are we doing wrong?

Is it my hosting provider? (I don't think so, I had already sent emails to @icloud in earlier versions of PHPMailer)
 
here is my code:

```
  function sendMail($umail,$message,$subject) {
    try {
      require_once (ROOT."/assets/resources/phpmailer/PHPMailer.php");
      require_once (ROOT."/assets/resources/phpmailer/SMTP.php");
      require_once (ROOT."/assets/resources/phpmailer/Exception.php");
      $mail = new PHPMailer\PHPMailer\PHPMailer(true);
      $mail->IsSMTP();
      $mail->SMTPDebug  = 0;
      $mail->SMTPAuth   = true;
      $mail->SMTPSecure = "tls";
      $mail->Host       = "leonpedro.me";
      $mail->Port       = 587;
      $mail->AddAddress($umail);
      $mail->Username="register_users@leonpedro.me";
      $mail->Password="*******";
      $mail->SetFrom("noreply@leonpedro.me","asdsdasdasda");
      $mail->Subject = $subject;
      $mail->MsgHTML($message);
      $mail->Send();
      return true;
    } catch (Exception $e) {
      return false;
    }
  }
```

**Comment by Synchro** [comment]
You need to take this up with iCloud support - you're doing everything right that I can see. Perhaps double-check your mail server's logs to be sure. It may help to update your DNS so that your hostname resolves backwards as well as forwards. It's currently like this:

```shell
# host leonpedro.me                                                                                                                                      
leonpedro.me has address 66.70.177.148
leonpedro.me mail is handled by 0 leonpedro.me.
# host 66.70.177.148
148.177.70.66.in-addr.arpa domain name pointer hades.e-novanet.mx.
```

Your SPF records looks OK.

**Comment by Synchro** [comment]
There was a workaround for an iCloud bug released in 6.0.1 today - you might want to give that a try.
